tests/input/array_indentation.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+$array = [
+0,
+ 1,
+  2,
+   3,
+    4,
+     5,
+];
+
+$multiArray = [
+    4 => [
+       7,
+        8,
+         9,
+    ],
```

Comment:
Please update the tests with a example with `string`s in the keys, like:
```php
$foo = [
'foo'   =>  [
  'bar' => 2,
1 => 'baz',
];
];
```

Reply:
Maybe this is also an interesting scenario:

```php
$items->append($one)
      ->append(
            [
0,
 1,
  2,
   ...,
              14,
               15,
                16,
            ]
      );
```

And I'd expect to have something like:


```php
$items->append($one)
      ->append(
            [
                0,
                1,
                2,
                ...,
                14,
                15,
                16,
            ]
      );
```

Reply:
@deeky666 that makes sense =)